src/Npc.js

```diff
@@ -99,6 +99,27 @@ class Npc extends Character {
       this.addItem(newItem);
     });
 
+    // Hydrate equipment
+    if (this.defaultEquipment.length > 0 && !(this.defaultEquipment instanceof Map)) {
+      const eqDefs = this.defaultEquipment;
+      this.defaultEquipment = new Map();
+      for (const slot in eqDefs) {
```

Comment:
Because that check isn't necessary this loop can just be

```suggestion
       for (const [slot, itemRef] of Object.entries(this.defaultEquipment)) {
```

Note that this says `itemRef` and not `itemDef` because the value is a _string_ entityReference, not an object definition like it is for players

Reply:
So where is slot coming from then?

Reply:
ah I see now!